### ISSUE TITLE
Accuracy with or without finetune

### ISSUE THREAD

**Issue by AdventureStory** [kekmodel/MPL-pytorch#35]
Hi, thanks for your great code, I am confused about accuracy with or without finetune on CIFAR-10-4K.
Results show that accuracy with and without finetune on CIFAR-10-4K are 96.01% and  96.08%, which means there is no significant increase of accuracy with finetune. And I wonder is it means we could use only unlabeled data (with pseudo label generated by teacher model) to train our student model and obtain accuracy almost equal to train it with labeled data? So it indicates indirectly that pseudo labels generated by teacher model are high-quality?
 I would appreciated if you could give me a response. Thanks!

**Comment by wowotoushiwo** [comment]
I am also puzzled by this question.

**Comment by kekmodel** [comment]
> So it indicates indirectly that pseudo labels generated by teacher model are high-quality?
	
This is because all training data is used for unlabeled data. I agree with this opinion.
